test/busted/outputHandlers/nvim.lua

```diff
@@ -1,10 +1,13 @@
 local pretty = require 'pl.pretty'
 local global_helpers = require('test.helpers')
 
--- Colors are disabled by default. #15610
 local colors = setmetatable({}, {__index = function() return function(s) return s == nil and '' or tostring(s) end end})
 if os.getenv "TEST_COLORS" then
-  colors = require 'term.colors'
+  local test_colors = os.getenv("TEST_COLORS"):lower()
+  local enable_colors = test_colors == 'true' or test_colors == '1' or test_colors == 'yes' or test_colors == 'on'
```

Comment:
Isn't it better to check for `false`, `0` or `off`? Anything else means enabled.